docs/environment-variables.md

```diff
@@ -0,0 +1,55 @@
+# Environment variables
+
+In development, environment variables are resolved from the .env file
+In production you are expected to set environment variables during deployment (e. g. using Vault)
+
+## List of environment variables
+
+### app
+
+- `NODE_ENV` - app execution mode. Supported values: `production` | `development` | `test`
+- `APP_ENV` - environment, in which app is running. Supported values: `production` | `development` | `staging`
+- `APP_BIND_ADDRESS` - address, on which server will be listening for HTTP(S) connections. e. g. `0.0.0.0`
+- `LOG_LEVEL` - logs starting from which level should be emitted. Supported values: `fatal` | `error` | `warn` | `info` | `debug` | `trace` | `silent`
+- (OPTIONAL) `APP_PORT` - port, on which server will be listening for HTTP(S) connections (`3000`)
```

Comment:
@nediml Which port do we currently use in prod?

Reply:
`3000`